src/converse-register.js

```diff
@@ -349,6 +372,9 @@
                         $form.append('<input type="button" class="submit" value="'+__('Return')+'"/>');
                         $form.find('input[type=button]').on('click', this.cancelRegistration.bind(this));
                     }
+                    if(_converse.registration_domain !== '') {
+                        $form.find('input[type=button]').hide();
+                    }
```

Comment:
This is the registration form rendered. The cancel button basically allows the end user to go back to the form where they can specify the registration domain. Since, there is no requirement to go back to that form, we don't need this button right?